packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -103,8 +103,15 @@ class TimeInput extends React.Component {
     let meridiem;
 
     if (props.variant === TimeUtil.FORMAT_12_HOUR) {
-      this.anteMeridiem = context.intl.formatMessage({ id: 'Terra.timeInput.am' });
-      this.postMeridiem = context.intl.formatMessage({ id: 'Terra.timeInput.pm' });
+      if (!context.intl.messages['Terra.timeInput.am'] || !context.intl.messages['Terra.timeInput.pm']) {
+        console.warn('This locale only uses 12 hour clock. The ante meridiem and post meridiem will not be displayed');
```

Comment:
Is a console.warn something we want to add?

Reply:
I think we need to give some kind of heads up to the developer, like we did up above for when an invalid input was provided to this package.

Reply:
If there is concern of this warning appearing in prod, we could wrap it in a check so it only shows the warning when webpack is run with non prod flag.

Reply:
Overlooked that we did that in the time-input with other warnings. Added it in the following commit.

https://github.com/cerner/terra-core/pull/1203/commits/2885e321bb44b28ee4ebc59c4aa3b4c7332132bf